code/frameworks/angular/src/server/framework-preset-angular-ivy.ts

```diff
@@ -58,7 +58,7 @@ export const webpack = async (webpackConfig: Configuration, options: PresetOptio
     return webpackConfig;
   }
 
-  if(angularOptions.enableNgcc !== false) {
+  if (angularOptions.enableNgcc !== false) {
```

Comment:
This seems unrelated to this PR, but I'm guessing it'll fall back out when `next` is updated and merged in.